src/v2/BaseLoginRouter.js

```diff
@@ -207,25 +222,32 @@ export default Router.extend({
         });
     }
 
-    // introspect stateToken when widget is bootstrap with state token
-    // and remove it from `settings` afterwards as IDX response always has
-    // state token (which will be set into AppState)
+    // Remove stateToken and other attributes to sanitize settings object.
+    // IDX response always has stateToken, that will be set into AppState)
+    const renderCallback = (idxResp) => {
+      this.unsetAttributes();
+      if (!idxResp.error) {
+        this.appState.trigger('remediationSuccess', idxResp);
```

Comment:
@haishengwu-okta should i clear the storage here for all terminal messages ? instead of the view ? Wondering there are no other flow then terminal ends up here..